packages/editor/src/editor-styles.scss

```diff
@@ -1,19 +1,113 @@
+/**
+ * Normalization Styles
+ *
+ * Why do these exist? Why not rely on browser defaults?
+ * These styles are necessary so long as CSS can bleed from the wp-admin into the editing canvas itself.
+ *
+ * Let's continue working to refactor these away, whether through Shadow DOM or better scoping of upstream styles.
+ */
+
 body {
 	font-family: $editor-font;
 	font-size: $editor-font-size;
 	line-height: $editor-line-height;
 	color: $dark-gray-900;
 }
 
+/* Headings */
+// These follow a Major Third type scale
```

Comment:
Is there a useful link for this?

Reply:
Context in #5631. Link: https://type-scale.com/?size=16&scale=1.250&text=A%20Visual%20Type%20Scale&webfont=&font-family=%27Noto%20Serif%27,%20serif&font-weight=600&font-family-headers=&font-weight-headers=inherit&background-color=white&font-color=